packages/suite/src/support/messages.ts

```diff
@@ -5443,7 +5443,7 @@ export default defineMessages({
         id: 'AMOUNT_IS_TOO_LOW',
     },
     AMOUNT_IS_BELOW_DUST: {
-        defaultMessage: 'Amount must be greater than the dust limit ({dust})',
+        defaultMessage: 'Amount must be greater than or equal the dust limit ({dust})',
```

Comment:
Has to be applied in crowding, otherwise this change gets lost on next sync. I'll change it there...

Reply:
done